libevmasm/Assembly.h

```diff
@@ -154,6 +154,12 @@ class Assembly
 		bool _includeSourceList = true
 	) const;
 
+	/// Loads the JSON representation of assembly.
+	/// @param _json JSON object containing assembly
+	/// @param _loadSources true, if source list should be included, false otherwise.
+	/// @returns true on success, false otherwise
+	static std::shared_ptr<Assembly> loadFromAssemblyJSON(Json::Value const& _json, std::vector<std::string> const& _sourceList = {}, bool _isCreation = true);
```

Comment:
This docstring no longer matches the declaration. There's no parameter called `_loadSources` and the function does not return `bool`.

Reply:
By the way, it's long enough that it could use some wrapping:
```suggestion
	/// @param _loadSources true, if source list should be included, false otherwise.
	/// @returns true on success, false otherwise
	static std::shared_ptr<Assembly> loadFromAssemblyJSON(
		Json::Value const& _json,
		std::vector<std::string> const& _sourceList = {},
		bool _isCreation = true
	);
```